### DISCUSSION TITLE
Unable to see anythig in my feed

### DISCUSSION THREAD

**Discussion by rahuldshetty** [community/community#67287] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I launch my main feed, it keeps failing with this error:

`There was an error in loading the activity feed. [Reload this page](https://github.com/).`

Following API calls failed with 502 status code: https://github.com/conduit/for_you_feed

**Comment by bwkam** [comment]
same

**Reply by JeanTheFungle** [reply]
same

**Comment by hoothin** [comment]
same

**Comment by JeanTheFungle** [comment]
what happend to github today ??

**Comment by ramin-guliyev** [comment]
Same issue

**Comment by bwkam** [comment]
Works again for me now.

**Comment by stefan-it** [comment]
It's time to bring back old feed!
